applications/nrf_desktop/doc/dfu.rst

```diff
@@ -77,6 +77,19 @@ These values are fetched using the :ref:`devinfo <dfu_devinfo>` configuration ch
    By default, the reported Vendor ID, Product ID, and generation are aligned with the values defined globally for the nRF Desktop application.
    The default values of Kconfig options used by the DFU module are based on respectively :ref:`CONFIG_DESKTOP_DEVICE_VID <config_desktop_app_options>`, :ref:`CONFIG_DESKTOP_DEVICE_PID <config_desktop_app_options>` and :ref:`CONFIG_DESKTOP_DEVICE_GENERATION <config_desktop_app_options>`.
 
+Flash access synchronization with other DFU methods
+===================================================
+
+The DFU module leverages the :ref:`nrf_desktop_dfu_lock` to synchronize flash access with other DFU methods (e.g. SMP DFU).
+Please use the :ref:`CONFIG_DESKTOP_DFU_LOCK <config_desktop_app_options>` option to activate this behaviour.
+
+On each DFU attempt, the module attempts to claim ownership over the DFU flash using the DFU Lock API.
+It holds the DFU owner status until the DFU process is complete or timed out.
```

Comment:
```suggestion
It holds the DFU owner status until the DFU process is completed or timed out.
```
since we're talking about process (unified in the other place as well)